### DISCUSSION TITLE
Difficult to answer path surface for crosswalks

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#3713] [Feedback]
https://www.openstreetmap.org/note/2986285#map=15/42.3217/-72.6895

<img src="https://user-images.githubusercontent.com/6563664/152273979-cd869fcf-4dbd-4736-84c1-1e41967169da.png" width="300" alt="SC View" /> <img src="https://user-images.githubusercontent.com/6563664/152272166-c91f8ec5-266a-43f9-9382-703f9e9e29e9.jpg" width="300" alt="picture" />

Note: The picture doesn't quite show the full highlighted segment.

It's hard to answer because:

- On one hand, the majority of the highlighted section is asphalt. So this is one logical answer.
- On the other hand, the road is already tagged as asphalt, and the parts of this segment of path which are not on the road are concrete.
- I don't think it makes sense to split the way here. It's already a small segment.

**Comment by matkoniecz** [comment]
I am usually splitting.

**Reply by Dimitar5555** [reply]
I'm also for splitting. Mostly because it allows you to map the curbs between the crossing and the sidewalk.
![image](https://user-images.githubusercontent.com/19364673/152295957-c9a44314-81fd-48a4-8dc5-f4148535a9c7.png)

**Reply by smichel17** [reply]
Isn't curb information already asked/recorded in a different SC quest?

I'll be honest: splitting does not seem that appealing to me, since this is a pretty common occurrence for crosswalks, and the split requires high accuracy on a small stretch of path. That said, I'll try it next time and see what I think.

**Reply by Dimitar5555** [reply]
There is quest for it: "Are there curbs where this way meets this road here? What kind?", but I'm not a big fan of adding kerb data on the crossing node,
Actually it doesn't require that high accuracy. It's enough to be able to tell the difference between the kerb and the road which is achievable in most cities with Esri Clarity or Mapbox.

**Comment by BalooUriza** [comment]
The way should be split but this might be a job for a different editor than
something in the field, given the precision involved.

**Reply by matkoniecz** [reply]
SC is entirely enough to split, though later it may be refined if very precise aerials are available

**Comment by RubenKelevra** [comment]
Well the right approach is to ignore it in SC and split it with aerial shots on the curb. This allows you to also tag the curb. :)

**Comment by mnalis** [comment]
Disregarding separate related issue (i.e. if I would split the way _even if the surface quest was not being asked_ in order to correctly map the kerbs) which has potentially high usability for disabled persons routing (and which I'd usually - for better results - do in JOSM via satellite imagery instead of in SC)...

I usually just mark it as an asphalt. I don't really see the **use case** of such nano-mapping like an footway of asphalt interleaved with a half of meter of concrete at the crossing etc.

<details>

So I would only split it if I had a lot of free time - and then **only if  drastically different; e.g. one part was asphalt and other part dirt** (as that might _theoretically_ have some use - e.g. if one is _"going to a ball in a princess dress with white linen shoes and must avoid any possibility of mud, so they painstakingly write an app for such navigation in advance"_.  Of course prerequisite is sending your troops of minions to make sure all such data is mapped 100% correct and have not changed before the ball, and it is much easier for princess in white linen shoes to arrange people carry her over any mud, but hey, there is _theoretically possibility_ of such data to be useful :smile:  )

And if one would really be into nano-mapping (with disregard of usefulness of it, and ignoring growing problems with OSM database size), they'd surely also have to map tactile indication for blind persons as separate part of the way with separate surface. Actually, they might even think that is insufficiently precise representation of reality, and suggest that unclosed ways are a bummer and one should instead map everything as a surface area... 

Just my opinion, of course...

</details>

Actually, I'd quite prefer if certain quests (like `smoothness` and `surface`) were **_not asked by SC for such small segments_** especially at intersections (https://github.com/streetcomplete/StreetComplete/discussions/3626#discussioncomment-1956514) as it IMHO wastes users time much more than the possible usefulness of the answers could be.
